Annex60/Utilities/Time/Examples/CalendarTime.mo

```diff
@@ -0,0 +1,52 @@
+within Annex60.Utilities.Time.Examples;
+model CalendarTime
+  extends Modelica.Icons.Example;
+  // fixme: comment of calendarTime2016 should probably be changed
```

Comment:
@Mathadon The comment is currently

> ... assuming time=0 corresponds to new year 2016"

This can be changed because if we set `final Real monthRef=1` etc., then `time=0` is always a new year, or maybe you simply wanted to stress the year of 2016?